subprojects/gradle/critical-path/critical-path/src/gradleTest/kotlin/com/avito/android/critical_path/CriticalPathTest.kt

```diff
@@ -111,8 +111,7 @@ internal class CriticalPathTest {
         // no op
     }
 
-    // TODO: this test is flaky but enabled to collect more info
-    //  If it's failed, please add info to MBS-11302
+    @Disabled("Fails constantly")
```

Comment:
This test is already deleted in #1110

Reply:
i don't see it in develop so it's merged ok